src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final String[] COLORS = new String[] {"black", "white", "yellow",
+            "blue", "red", "pink", "green", "gray"};
+
+    public String randomColor() {
```

Comment:
method names should be verbs
```suggestion
    public String getRandomColor() {
```